cenc-format.html

```diff
@@ -302,23 +301,16 @@
   </p>
   <h1 class="title p-name" id="title" property="dcterms:title">ISO Common Encryption EME Stream Format and Initialization Data</h1>
   
-  <h2 id="w3c-editor-s-draft-10-december-2015"><abbr title="World Wide Web Consortium">W3C</abbr> Editor's Draft <time property="dcterms:issued" class="dt-published" datetime="2015-12-10">10 December 2015</time></h2>
+  <h2 id="w3c-document-16-february-2016"><abbr title="World Wide Web Consortium">W3C</abbr> Document <time property="dcterms:issued" class="dt-published" datetime="2016-02-16">16 February 2016</time></h2>
   <dl>
     
-      <dt>This version:</dt>
-      <dd><a class="u-url" href="https://w3c.github.io/encrypted-media/cenc-format.html">https://w3c.github.io/encrypted-media/cenc-format.html</a></dd>
-      <dt>Latest published version:</dt>
-      <dd><a href="http://www.w3.org/TR/encrypted-media/">http://www.w3.org/TR/encrypted-media/</a></dd>
-    
     
       <dt>Latest editor's draft:</dt>
-      <dd><a href="https://w3c.github.io/encrypted-media/cenc-format.html">https://w3c.github.io/encrypted-media/cenc-format.html</a></dd>
-    
+      <dd><a href="https://www.w3.org/html/media/cenc-format.html">https://www.w3.org/html/media/cenc-format.html</a></dd>
```

Comment:
The use of specStatus: "base" in ReSpec causes there to be no latest-but-not-yet-published URL.
a) Will these be auto-published?
b) Will they always be auto-published even after EME moves to CR such that there is no need for a "latest" URL?